examples/gaussian-process-priors/script.jl

```diff
@@ -120,6 +120,7 @@ kernels = [
     LinearKernel(),
     compose(PeriodicKernel(), ScaleTransform(0.2)),
     NeuralNetworkKernel(),
+    GibbsKernel(x -> exp(sin.(x))),
```

Comment:
Usually we prefer the keyword argument constructor since it is a bit clearer. You can also simplify the lengthscale function a bit (broadcasting is only useful if `x` are vectors - in which case `exp(sin.(x))` would fail though):
```suggestion
    GibbsKernel(; lengthscale=exp ∘ sin),
```

Reply:
I think we should generally handle vectors so something like `exp(sum(sin, x))` maybe?

Reply:
It seemed the example only deals with scalar inputs but maybe I didn't check it carefully enough. If vectors occur as inputs, then I agree something like `x -> exp(sum(sin, x))` or `x -> sum(exp ∘ sin, x)` should be used.

Reply:
Thanks both for the helpful information and suggestions. I went with `x -> sum(exp ∘ sin, x)` and checked locally that it works in a notebook so hopefully the CI will be happy.